src/main/kotlin/lotto/domain/LotteryWinningTypes.kt

```diff
@@ -0,0 +1,16 @@
+package lotto.domain
+
+import lotto.domain.ExceptionType.NOT_DEFINED_HIT_FOR_WINNINGS
+
+enum class LotteryWinningTypes(val hits: Int, val winnings: Int) {
+    Three(3, 5000),
+    Four(4, 50000),
+    Five(5, 1500000),
+    Six(6, 2000000000);
```

Comment:
```suggestion
    Six(6, 2_000_000_000);
```
언더스코어를 활용하면 가독성을 높힐 수 있습니다!